app/db.py

```
@@ -0,0 +1,91 @@
+            rows.append({
+                "event_id": ev.id,
+                "kind": ev.kind,
+                "user_email": ev.user.email if ev.user else None,
+                "created_at": ev.created_at.isoformat(),
+            })
+    return rows
+
+def add_funds_float(user_id: int, delta: float) -> float:
+    with get_session() as s:
+        u = s.get(User, user_id)
+        if not u: return 0.0
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)